### PR TITLE
Removed ISIS3DATA from base apps

### PR DESCRIPTION
**Pull request by scsides**
Removed ISIS3DATA from base apps

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/apps/maptemplate/maptemplate.xml [hunk 1]
```diff
@@ -91,7 +91,7 @@
        it requires keywords that are not defined in the program.<br /><br />
      </li>
      <li>
-       Templates in $ISIS3DATA/base/templates/maps:  the template contains all the 
+	 Templates in $ISISROOT/appdata/templates/maps:  the template contains all the
```
Comment (krlberry) [inline isis/src/base/apps/maptemplate/maptemplate.xml]: maps is actually at `$ISISROOT/appdata/maps`

File: isis/src/base/apps/maptemplate/maptemplate.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@
      To convert a raw instrument/camera cube (<def>Level1</def>) to a map projected 
      image (<def>Level2</def>) the user must (1) create a  custom <def>PVL</def> formatted 
      text file, (2) use this <i>maptemplate</i> application to produce one, or (3) use 
-     the automated, projection-specific Map File Template in $ISIS3DATA/base/templates/maps.
+       the automated, projection-specific Map File Template in $ISISROOT/appdata/templates/maps.
```
Comment (krlberry) [inline isis/src/base/apps/maptemplate/maptemplate.xml]: maps is actually at $ISISROOT/appdata/maps

File: isis/src/tgo/objs/TgoCassisCamera/TgoCassisDistortionMap.cpp [hunk 3]
```diff
@@ -109,7 +109,7 @@ namespace Isis {
     // -1/2 * pixel pitch * CCD height = -10.24 < y < 10.24 = 1/2 * pixel pitch * CCD height
     // 
     // Also, the zeros for the divider variable fall well outside the boundary
-    // of the CCD. (See $ISIS3DATA/tgo/assets/distortion/DistortionModelA3CorrRoots.jpg).
+    // of the CCD. (See $ISISDATA/tgo/assets/distortion/DistortionModelA3CorrRoots.jpg).
```
Comment (krlberry) [inline isis/src/tgo/objs/TgoCassisCamera/TgoCassisDistortionMap.cpp:112]: Agree with update based on where the data is located right now, but should we potentially move the files containing the details of the CaSSIS distortion model out of the ISIS repository as discussed? I'm still not sure where to put these.

File: isis/src/base/apps/smtk/smtk.xml [hunk 4]
```diff
@@ -353,7 +353,7 @@
             <function>helperButtonLog</function>
             <brief>View file in log area</brief>
             <description>This helper button will display the regdef file in the log area</description>
-            <icon>$ISIS3DATA/base/icons/view_text.png</icon>
+            <icon>$ISISDATA/base/icons/view_text.png</icon>
```
Comment (krlberry) [inline isis/src/base/apps/smtk/smtk.xml]: `$ISISROOT/appdata/images/icons`

File: isis/src/base/objs/Equalization/Equalization.truth [hunk 5]
```diff
@@ -8546,13 +8546,13 @@ Testing loadInputs exception ...
 **USER ERROR** The input file [HoldList.lst] must contain at least 2 file names.
 
 Testing errorCheck exception (1)...
-**USER ERROR** Number of bands do not match between cubes [data/odyssey/testData/I00824006RDR.lev2.cub] and [data/base/testData/isisTruth.cub].
+**USER ERROR** Number of bands do not match between cubes [odyssey/testData/I00824006RDR.lev2.cub] and [/usgs/cpkgs/isis3/isis_data/base/testData/isisTruth.cub].
```
Comment (krlberry) [inline isis/src/base/objs/Equalization/Equalization.truth]: I think this needs to be changed to not have the full path for the test to succeed.

Comment (krlberry) [reply isis/src/base/objs/Equalization/Equalization.truth]: Is this missing a regex to strip out the path, or is the regex not working?

File: isis/src/base/objs/Pipeline/unitTest.xml [hunk 6]
```diff
@@ -231,7 +231,7 @@
             <option value="SYSTEM">
               <brief>Search system for a default shape model</brief>
               <description>
-                This option will search the default system data area ($ISIS3DATA/base/dems) for a shape model that 
+                This option will search the default system data area ($ISI3DATA/base/dems) for a shape model that
```
Comment (krlberry) [inline isis/src/base/objs/Pipeline/unitTest.xml]: typo: $ISI3DATA -> $ISISDATA

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml [hunk 7]
```diff
@@ -83,22 +83,22 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
       <li>Finally, setup the environment variables:
```
Comment (krlberry) [inline isis/src/docsys/documents/InstallGuide/InstallGuide.xml:83]: The updates to this file make sense, but I don't think there's any way to access this page via https://isis.astrogeology.usgs.gov/Installation/index.html any more and I don't think it's being kept up to date.

**Review by krlberry — CHANGES_REQUESTED** [review]
Found some minor typos and incorrect paths.

File: isis/src/tgo/objs/TgoCassisCamera/TgoCassisDistortionMap.cpp (hunk 3 above)
Comment (scsides) [reply isis/src/tgo/objs/TgoCassisCamera/TgoCassisDistortionMap.cpp:112]: This is on the todo list. We need to find a good place to keep these type of assets

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml (hunk 7 above)
Comment (scsides) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml:83]: Agreed, we should consider removing a lot of the documentation I think

**Pull request merged**